.golangci.yaml

```diff
@@ -0,0 +1,20 @@
+run:
+  timeout: 5m
+linters:
+  disable-all: true
+  enable:
+    - gosimple
+    - govet
+    - staticcheck
+    - unused
+    - revive
```

Comment:
I believe revive doesn't lint anything by default. I believe you're going to have to copy the rules from the revive.toml into a linter-settings format unfortunately https://golangci-lint.run/usage/linters/#revive

Reply:
I checked the revive docs and assumed the opposite.

> Please notice that if no particular configuration is provided, revive will behave as go-lint does, i.e. all go-lint rules are enabled (the [Available Rules table](https://github.com/mgechev/revive?tab=readme-ov-file#available-rules) details what are the go-lint rules). When a configuration is provided, only rules in the configuration are enabled.

https://github.com/mgechev/revive?tab=readme-ov-file#golangci-lint

Reply:
Fixed this by enabling rules that are by default ignored by golangci-lint.

https://golangci-lint.run/usage/false-positives/#exc0012